helper/testhelpers/testhelpers.go

```diff
@@ -265,10 +265,6 @@ func ConfClusterAndCore(t testing.T, conf *vault.CoreConfig, opts *vault.TestClu
 		"approle":  credAppRole.Factory,
 		"userpass": credUserpass.Factory,
 	}
-	coreConfig.LogicalBackends = map[string]logical.Factory{
```

Comment:
Instead of completely removing, maybe do a nil check and set these as the logical backends if none are provided.

Reply:
They're completely specific to two tests (one each).  There's no reason to provide them outside of those.

Reply:
Ahh, I see.